setup.py

```diff
@@ -7,8 +7,8 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        'FeinCMS>=1.7',
-        'django-imagekit==3.2.1',
+        'FeinCMS>=1.7,<2',
```

Comment:
Given that django 1.7 is only supported in FeinCMS >= 1.10, I think we should be going for `>=1.10,<1.11`